src/openklant/components/contactgegevens/migrations/0004_alter_organisatie_adres_land_alter_organisatie_land_and_more.py

```diff
@@ -0,0 +1,134 @@
+# Generated by Django 4.2.15 on 2025-01-09 15:19
+import logging
+import django.core.validators
+from django.db import migrations, models
+import openklant.utils.validators
+from django.core.management import CommandError
+
+from openklant.utils.constants import COUNTRIES_DICT
+
+logger = logging.getLogger(__name__)
+
+FIELDS = ["adres_land", "land"]
+
+
+def _check_records(records, model_name):
+    total_failed_records = 0
+    for record in records:
+        record_failed = False
+        for field_name in FIELDS:
+            field_value = getattr(record, field_name, None)
+            iso_code = COUNTRIES_DICT.get(field_value, "")
+            if field_value and not iso_code:
+                record_failed = True
+                logger.warning(
+                    "%s(pk=%s, uuid=%s) Field: '%s'. No match found for nl_code: '%s'",
+                    model_name,
+                    record.pk,
+                    record.uuid,
+                    field_name,
+                    field_value,
+                )
+
+        if record_failed:
+            total_failed_records += 1
+    return total_failed_records
+
+
+def _update_records(records):
+    for record in records:
+        updated = False
+        for field_name in FIELDS:
+            if field_value := getattr(record, field_name, None):
+                iso_code = COUNTRIES_DICT.get(field_value, "")
+                setattr(record, field_name, iso_code)
+                updated = True
+
+        if updated:
+            record.save()
+
+
+def _check_and_update_records(apps, schema_editor):
+    Organisatie = apps.get_model("contactgegevens", "Organisatie")
+    Persoon = apps.get_model("contactgegevens", "Persoon")
+
+    for model_name, model in [("Organisatie", Organisatie), ("Persoon", Persoon)]:
```

Comment:
I think you could use `model.__qualname__` to get the classname as a string, if that works you can remove the hardcoded classname

```suggestion
    for model in (Organisatie, Persoon,):
```